manifests/extension/uopz.pp

```diff
@@ -0,0 +1,58 @@
+# Installs a php extension for a specific version of php.
+#
+# Usage:
+#
+#     php::extension::uopz { 'uopz for 5.4.10':
+#       php     => '5.4.10',
+#       version => '1.0.3'
+#     }
+#
+define php::extension::uopz(
+  $php,
+  $version = '2.0.6'
+) {
+  include boxen::config
+
+  require php::config
+  # Require php version eg. php::5_4_10
```

Comment:
This comment uses the old syntax. Can you please update to the new syntax or remove it to avoid confusion?